sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/rest/SormasToSormasRestClient.java

```diff
@@ -186,6 +188,7 @@ private <T> T handleResponse(Response response, Class<T> responseType) throws So
 			String errorI18nTag = null;
 			List<ValidationErrors> errors = null;
 			Object[] args = null;
+			boolean debug = false;
```

Comment:
Is this debug code which can be removed?

Reply:
I also think we can delete this flag

Reply:
ok